src/test/twirl/uk/gov/hmrc/play/views/layouts/test_includes_that_use_config.scala.html

```diff
@@ -0,0 +1,30 @@
+@import uk.gov.hmrc.play.config.OptimizelyConfig
```

Comment:
IMO it feels like the name of the file `test_includes_that_use_config.scala.html` isn't very clear and could just be `test_header_footer_includes.scala.html` and then the comment in the file explains the particulars